packages/api/helper/nominationGsheetToDB.js

```diff
@@ -3,19 +3,14 @@ const { google } = require('googleapis');
 const db = require('../models');
 const parsePrivateKey = require('./parsePrivateKey');
 const rangePar = 'Sheet1';
-const clientEmail = process.env.GOOGLE_SHEETS_CLIENT_EMAIL
-const privateKey = parsePrivateKey(process.env.GOOGLE_SHEETS_PRIVATE_KEY)
-const scopes = ['https://www.googleapis.com/auth/spreadsheets']
+const clientEmail = process.env.GOOGLE_SHEETS_CLIENT_EMAIL;
+const privateKey = parsePrivateKey(process.env.GOOGLE_SHEETS_PRIVATE_KEY);
+const scopes = ['https://www.googleapis.com/auth/spreadsheets'];
 const { verifyHcEmail } = require('./mailer.js');
 
 module.exports = function gsheetToDB() {
-  const client = new google.auth.JWT(
-    clientEmail,
-    null,
-    privateKey,
-    scopes
-  );
-
+  const client = new google.auth.JWT(clientEmail, null, privateKey, scopes);
+  let domains;
```

Comment:
Is this variable being used anywhere?

Reply:
I checked it and it is not. I deleted it.